tools/linter_lib/custom_check.py

```diff
@@ -484,6 +484,11 @@ def build_custom_checkers(by_lang):
          ]),
          'description': 'Comment-style function type annotation. Use Python3 style annotations instead.',
          },
+        {'pattern': 'null=True',
+         'exclude_pattern': '# type: Optional',
+         'include_only': {"zerver/models.py"},
+         'description': 'Model variable with null=true not annotated as Optional.',
```

Comment:
Can you add a `good_lines` and `bad_lines` to test this new rule?

We're sorta working on a goal of having all of our linter rules have tests.